daemon/services/collectors/filewatcher.go

```
@@ -15,10 +15,11 @@ import (
 // It provides debouncing to coalesce rapid successive fs events (e.g. truncate+write)
 // into a single callback invocation.
 type FileWatcher struct {
-	watcher  *fsnotify.Watcher
-	mu       sync.Mutex
-	debounce time.Duration
-	timers   map[string]*time.Timer
+	watcher   *fsnotify.Watcher
```

golangci-lint (2.12.2)

[error] File is not properly formatted

(gofmt)